### DISCUSSION TITLE
Info box 要如何添加？

### DISCUSSION THREAD

**Discussion by jinyiliu** [ppoffice/hexo-theme-icarus#949] [Q&A] [answered]
没有找到应该如何添加一个提示、警告的 Info box ，就是下面这个

<img width="631" alt="截屏2021-10-07 下午2 55 01" src="https://user-images.githubusercontent.com/23398808/136334459-da514ce8-43a2-4931-a6ea-6870e49eb5b3.png">

想问一下怎么添加？需要另外安装别的插件吗？

**Comment by Mrzhang06** [comment, marked as answer]
直接写在你的md文档中
```HTML warning-info
<article class="message message-immersive is-warning">
  <div class="message-body">
    <i class="fas fa-question-circle mr-2"></i>文章内容有误？请点击 
    <a href="你自己接收反馈的URL">此处</a>提交修改。
  </div>
</article>
```

**Reply by JIeJaitt** [reply]
有其他写法吗？

**Reply by Mrzhang06** [reply]
前端不熟，这是在iMaeGoo大佬博客评论里面看到的

**Reply by JIeJaitt** [reply]
> 前端不熟，这是在iMaeGoo大佬博客评论里面看到的

哪篇文章的评论？方便给个链接吗？

**Reply by Mrzhang06** [reply]
https://www.imaegoo.com/2020/icarus-with-bulma/

评论里面问的跟这个基本一样，看了文档页源码，就是变了个class名和图标
